kotlin/legacy/legacy-workflow-core/src/main/java/com/squareup/workflow/legacy/WorkflowOperators.kt

```diff
@@ -113,6 +115,7 @@ fun <S : Any, E : Any, O1 : Any, O2 : Any> Workflow<S, E, O1>.mapResult(
   // Propagate cancellation upstream.
   transformedResult.invokeOnCompletion { cause ->
     if (cause != null) {
+      @Suppress("DEPRECATION")
```

Comment:
Worth noting why we can't use the un-deprecated path, if there is one.

Reply:
Filed #188 and added TODOs.